### PR TITLE
Updates for RDKit 2020 changes

### PR DESCRIPTION
**Pull request by mattwthompson**
## Description
This is kinda cheating, as it's stacking the deck with the expected RDKit output. But I'm not sure that's the wrong thing to do. There are numerous in-the-weeds changes in RDKit 2020, but we don't need to capture them unless we want to safely upconvert from the result of a CMILES using RDKit 2019 - something I'd like to avoid as much as possible.

This is all using 2020.03.6, although there should be a 2020.09.something version coming down the pipes soon.

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [ ] TODO 1

## Questions
- [ ] Question1

## Status
- [ ] Ready to go

### CODE REVIEW

**Comment by mattwthompson** [comment]
Copying an important detail in one of the commit messages:

> Note that OpenEye's behavior appears to be the same with the 2020
versions released so far, so no other changes should be necessary

I was just copying the file around to make it follow a similar structure to the RDKit tests

File: cmiles/tests/reference/generate_reference.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # Regenerate with molecule names (makes it easier to search later)
 
 # Make sure to add SMILES to input smi file. RDKit expects that line there and just skips the first line
-mols = rdkit.Chem.SmilesMolSupplier('drug_bank_stereo.smi')
+mols = rdkit.Chem.SmilesMolSupplier('drug_bank_stereo.smi', titleLine=False)
```
Comment (j-wags) [inline cmiles/tests/reference/generate_reference.py:8]: (not blocking) did the previous results file not have this title line by default? I'm wondering if RDKit just recently changed this behavior?

**Review by j-wags — APPROVED** [review]
Excellent. LGTM!

File: cmiles/tests/reference/generate_reference.py (hunk 1 above)
Comment (mattwthompson) [reply cmiles/tests/reference/generate_reference.py:8]: Just checked with separate environments and it's not an RDKit change. I think this is something that was done by hand while running this script again - which I found frustrating, hence the change here.

**Comment by mattwthompson** [comment]
Thanks @j-wags!

**Pull request merged**